pages/common/postcss.md

```diff
@@ -0,0 +1,35 @@
+# postcss
+
+> PostCSS is a tool for transforming styles with JS plugins.
+
+- Parse and transform CSS file(s):
+
+`postcss {{path/to/file(s)}}`
+
+- Parse and transform CSS file(s) and output to a specific file:
+
+`postcss {{path/to/file(s)}} --output {{path/to/file}}`
```

Comment:
Can you check if multiple files can output to a single file ? This is what it says in the README ` If there are multiple input files, the --dir or --replace option must be passed.` https://github.com/postcss/postcss-cli#usage

Reply:
My bad, looks like that is correct.

I'll change them all to say `path/to/file`. 👍